accounts/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from django.contrib.auth.models import PermissionsMixin
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.core.mail import send_mail
+from django.utils.translation import ugettext_lazy as _
+from .managers import MyUserManager
+
+
+class MyUser(AbstractBaseUser, PermissionsMixin):
+    first_name = models.CharField(_('first name'), max_length=30, blank=True)
```

Comment:
We don't need to specify `first_name`, `last_name`, `is_active`, `email`, `is_staff` fields here, AbstractBaseUser provides us with them